### ISSUE TITLE
run “noop” operation with "-synchronous" option should return "got noop response"

### ISSUE THREAD

**Issue by Wayenyin** [Microsoft/omi#292]
**Good result:** 
Run command without "-synchronous": 
```
/opt/omi/bin/omicli noop
    got noop response
```

**Issue:**
Run command with "-synchronous": 
```
/opt/omi/bin/omicli noop -synchronous
```

Build：omi-1.2.0-76.x86_64

Actual: there is nothing output
Expect: output should be "got noop response"

**Comment by paulcallen** [comment]
@palladia can you take a look.

**Comment by palladia** [comment]
Fixed with #296
